### DISCUSSION TITLE
support@vocalvoter.com returns MAIL UNDELIVERABLE

### DISCUSSION THREAD

**Discussion by avincent98144** [community/community#50707] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

support@vocalvoter.com returns MAIL UNDELIVERABLE

please advise as my issue is even tho my correct zip code is entered, I get REPS from the other side of the country!

**Comment by otter-computer** [comment]
Hi there,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
